sencha-workspace/SlateTasksStudent/app/controller/Tasks.js

```diff
@@ -296,6 +209,49 @@ Ext.define('SlateTasksStudent.controller.Tasks', {
         }
 
         return filtered;
+    },
+
+    showDetails: function(rec) {
+        var me = this,
+            details = me.getTaskDetails(),
+            form = me.getTaskForm(),
+            ratingView = me.getRatingView(),
+            teacherAttachmentsField = me.getTeacherAttachmentsField(),
+            studentAttachmentsField = me.getStudentAttachmentsField(),
+            commentsField = me.getCommentsField(),
+            submissions = rec.get('Submissions'),
+            readonly = me.getTaskTree().getReadOnly() || rec.get('TaskStatus') === 'completed';
+
+        form.getForm().loadRecord(rec);
+
+        if (submissions && submissions.length && submissions.length > 0) {
+            form.down('displayfield[name="Submitted"]').hide();
```

Comment:
@BillClinton use the [`findField`](http://docs.sencha.com/extjs/6.2.1/classic/Ext.form.Basic.html#method-findField) method